TileFormats/FeatureTable/schema/i3dm.featureTable.schema.json

```diff
@@ -0,0 +1,71 @@
+{
+    "$schema" : "http://json-schema.org/draft-04/schema#",
+    "id" : "i3dm.featureTable.schema.json",
+    "title" : "Instanced 3D Model Feature Table",
+    "type" : "object",
+    "description" : "A set of Instanced 3D Model semantics that contains values defining the position and appearance properties for instanced models in a tile.",
+    "allOf" : [
+        {
+            "$ref" : "featureTable.schema.json"
+        },
```

Comment:
Minor whitespace here and throughout (but will save a level of indent), change lines like this to:
```
}, {
```

And above and throughout, can be like
```
[{
```

which are both what we do in JavaScript.